array-methods-more/only-all-females.js

```diff
@@ -4,6 +4,15 @@
     2. Add more tests to check this is working
  */
 
+const checkAllFemale = (persons) => {
```

Comment:
You forgot about the arrow function in the .every function!
See the description of .every function on MDN - [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every)
```javascript
const checkAllFemale = (persons) => {
    const result = persons.every((person) => {
        return person.gender === 'female'
    });
    return result;
};
```